components/layout_2020/style_ext.rs

```diff
@@ -953,6 +964,24 @@ impl ComputedValuesExt for ComputedValues {
         };
         has_percentage(box_offsets.block_start) || has_percentage(box_offsets.block_end)
     }
+
+    // <https://drafts.csswg.org/cssom-view/#overflow-directions>
+    fn scrolling_overflow_direction(&self) -> ScrollingOverflowDirection {
+        let writing_mode: WritingMode = self.writing_mode;
+        let inline_end_direction = writing_mode.inline_end_physical_side();
+        let block_end_direction = writing_mode.block_end_physical_side();
+
+        let rightward = inline_end_direction == PhysicalSide::Right ||
+            block_end_direction == PhysicalSide::Right;
+        let downward = inline_end_direction == PhysicalSide::Bottom ||
+            block_end_direction == PhysicalSide::Bottom;
+
+        // FIXME: We should consider the flex-container's CSS (e.g. flow-direction: column-reverse).
```

Comment:
Does Servo have a style guide about TODOs?

Anyway, it will be better to have assigned/tracked TODOs instead of anonymous ones.

For example: `TODO(stevennovaryo: We should consider blabla` or `TODO(#XXXXX): We should consider blabla`.

Reply:
I am not sure. I suppose it is much better to make the appropriate issue if it's clear enough. But I would like to know the details of flex and writing mode progress first.

Reply:
Direction and writing mode define the "logical axes", which are different from the physical (i.e., x & y) axes. See the "coordinate spaces" section of:

https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/layout/README.md#coordinate-spaces

Note that direction and writing mode never result in a bottom-to-top block direction.

Similar effect can only be achieve with flex layout using `flow-direction: column-reverse`